cypress/e2e/integration-manager/kick.spec.ts

```diff
@@ -0,0 +1,237 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { SynapseInstance } from "../../plugins/synapsedocker";
+import { MatrixClient } from "../../global";
+import { UserCredentials } from "../../support/login";
+
+const ROOM_NAME = "Integration Manager Test";
+const BOT_DISPLAY_NAME = "Bob";
+
+const INTEGRATION_MANAGER_TOKEN = "Sr_oU-B-Cir_Tx573Ugr";
+const INTEGRATION_MANAGER_HTML = `
+    <html lang="en">
+        <head>
+            <title>Fake Integration Manager</title>
+        </head>
+        <body>
+            <input type="text" id="target-room-id"/>
+            <input type="text" id="target-user-id"/>
+            <button name="Send" id="send-action">Press to send action</button>
+            <script>
+                document.getElementById("send-action").onclick = () => {
+                    window.parent.postMessage(
+                        {
+                            action: "kick",
+                            room_id: document.getElementById("target-room-id").value,
+                            user_id: document.getElementById("target-user-id").value,
+                            reason: "Removed from room",
+                        },
+                        '*',
+                    );
+                };
+            </script>
+        </body>
+    </html>
+`;
+
+function openIntegrationManager() {
+    cy.get(".mx_RightPanel_roomSummaryButton").click();
+    cy.get(".mx_RoomSummaryCard_appsGroup").within(() => {
+        cy.contains("Add widgets, bridges & bots").click();
+    });
+}
+
+function sendActionFromIntegrationManager(integrationManagerUrl: string, targetRoomId: string, targetUserId: string) {
+    cy.accessIframe(`iframe[src*="${integrationManagerUrl}"]`).within(() => {
+        cy.get("#target-room-id").should("exist").type(targetRoomId);
+        cy.get("#target-user-id").should("exist").type(targetUserId);
+        cy.get("#send-action").should("exist").click();
+    });
+    // Wait for the message to be handled
+    return cy.wait(100);
+}
+
+describe("Kick", () => {
```

Comment:
```suggestion
describe("Integration Manager: Kick", () => {
```

or surround in an outer `describe()`